packages/jest/package.json

```diff
@@ -59,5 +59,8 @@
 			"src/index.ts",
 			"src/environment.ts"
 		]
+	},
+	"engines": {
+		"node": ">=18.0.0"
```

Comment:
This seems like a major change since it will introduce breaking change?

Reply:
Done!